### ISSUE TITLE
Gui Is closed pressing E

### ISSUE THREAD

**Issue by SrRapero720** [CreativeMD/LittleFrames#55]
Typing or fixing a URL char with E as key closes the gui like inventory gui.
So if I want to write: gyazo.com/my-pro-pictur... Whoops, Gui was closed.

To fix this just overwrite onKeyPressed on gui
https://github.com/CreativeMD/LittleFrames/blob/b8494ecbcd1266d0288c37d9242011885a8e9c60/src/main/java/team/creative/littleframes/client/gui/GuiLittlePictureFrame.java#L33

Adding:
```java
    @Override
    public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
        if (keyCode == 256) {
            if (this.closeLayerUsingEscape()) {
                this.closeTopLayer();
            }
            return true;
        // Using "super.keyPressed()" also handles E, but with this is handled pressing E before CC handles it
        } else if (keyCode == this.getSettings().keyInventory.getKey().getValue()) {
            return true;
        } else if (super.keyPressed(keyCode, scanCode, modifiers)) {
            return true;
        } else {
            return false;
        }
    }
```

**Comment by CreativeMD** [comment]
Talking about 1.18 or newer versions?

**Comment by SrRapero720** [comment]
is affected 1.18 and 1.19
i didn't test on 1.20

**Comment by SrRapero720** [comment]
my bad, i have installed an old version of CreativeCore on 1.19.2
updating it solves it on 1.19.2 (new versions aren't afected)

**Comment by SrRapero720** [comment]
About that bug, can you Backport that CC fix to 1.18.2?

**Comment by CreativeMD** [comment]
Uploaded a fix for this just now!
